docs/guides/expectations/advanced/how-to-create-expectations-that-span-multiple-batches-using-evaluation-parameters.md

```diff
@@ -2,4 +2,147 @@
+
+<Prerequisites>
+
+- Configured a [Data Context](../../../tutorials/getting-started/initialize-a-data-context.md).
+- Configured a [Datasource](../../../reference/datasources) (or several Datasources) with at least two **Data Assets** and understand the basics of **Batch Requests**.
+- Also created [Expectations Suites](../../../tutorials/getting-started/create-your-first-expectations.md) for those Data Assets.
+- Have a working [Evaluation Parameter store](../../../reference/data-context#evaluation-parameter-stores). (The default in-memory store from ``great_expectations init`` can work for this.)
+- Have a working [Validation Operator](../../../reference/checkpoints-and-actions) (see also [How to add a Validation Operator](../../../guides/validation/how-to-add-a-validation-operator)). (The default Validation Operator from ``great_expectations init`` can work for this.)
+
+</Prerequisites>
+
+Steps
+-----
+
+In a notebook,
+
+1. **Import great_expectations and instantiate your Data Context**
+   ```python
+   import great_expectations as ge
+   context = ge.DataContext()
+   ```
+
+2. **Instantiate two Validators, one for each Data Asset**
+
+    We'll call one of these Validators the *upstream* Validator and the other the *downstream* Validator. Evaluation Parameters will allow us to use Validation Results from the upstream Validator as parameters passed into Expectations on the downstream.
+
+    It's common (but not required) for both Batch Requests to have the same [Datasource and Data Connector](../../../reference/datasources).
+
+    ```python
+    batch_request_1 = BatchRequest(
+        datasource_name="my_datasource",
+        data_connector_name="my_data_connector",
+        data_asset_name="my_data_asset_1"
+    )
+    upstream_validator = context.get_validator(batch_request=batch_request_1, expectation_suite_name="my_expectation_suite_1")
+
+    batch_request_2 = BatchRequest(
+        datasource_name="my_datasource",
+        data_connector_name="my_data_connector",
+        data_asset_name="my_data_asset_2"
+    )
+    downstream_validator = context.get_validator(batch_request=batch_request_2, expectation_suite_name="my_expectation_suite_2")
+    ```
+
+3. **Disable interactive evaluation for the downstream Validator.**
+
+    ```python
+    downstream_validator.interactive_evaluation = False
+    ```
+    Disabling interactive evaluation allows you to declare an Expectation even when it cannot be evaluated immediately.
+
+4. **Define an Expectation using an Evaluation Parameter on the downstream Validator.**
+
+    ```python
+        eval_param_urn = 'urn:great_expectations:validations:my_expectation_suite_1:expect_table_row_count_to_be_between.result.observed_value'
+        downstream_validator.expect_table_row_count_to_equal(
+            value={
+                '$PARAMETER': eval_param_urn, # this is the actual parameter we're going to use in the validation
+            }
+        )
+    ```
```

Comment:
I believe this should be left aligned (at least from what the Netlify build is showing me). Same with the below code block.

<img width="1042" alt="Screen Shot 2021-07-13 at 7 43 35 PM" src="https://user-images.githubusercontent.com/49923762/125539073-2ad09698-e037-4f55-9f70-6b70426bed1d.png">

Reply:
Will look and ping when ready.  Thanks.